.github/workflows/release.yml

```diff
@@ -0,0 +1,61 @@
+name: Create a Release
+
+on:
+  push:
+    tags:
+      - 'v*'
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Get the version
+        id: get_version
+        run: echo ::set-output name=VERSION::${GITHUB_REF#refs/tags/}
+
+      - name: REUSE Compliance Check
+        uses: fsfe/reuse-action@v1
+        with:
+          args: spdx -o zephyr-${{ steps.get_version.outputs.VERSION }}.spdx
+
+      - name: upload-results
+        uses: actions/upload-artifact@master
+        continue-on-error: True
+        with:
+          name: zephyr-${{ steps.get_version.outputs.VERSION }}.spdx
+          path: zephyr-${{ steps.get_version.outputs.VERSION }}.spdx
+
+      - name: Get Diff since last tag
+        run: |
+          oldtag=$(git describe --abbrev=0 ${{ github.ref }}^)
```

Comment:
This will include `-rc` tags, is that what you want?

Reply:
yes